framework/render/audio/IAudioRender.h

```diff
@@ -164,9 +165,17 @@ namespace Cicada{
             mListener = listener;
         }
 
+        virtual void setRenderingCb(onRenderFrame cb, void *userData)
+        {
+            mRenderingCb = cb;
+            mRenderingCbUserData = userData;
+        }
+
```

Comment:
should not set the rendering callback to filter

Reply:
set the callback in IAudioRender